api/src/main/resources/application.properties

```diff
@@ -5,6 +5,11 @@
 #spring.jpa.properties.hibernate.format_sql=true
 #spring.jpa.properties.hibernate.type=trace
 
+#Hibernate batching properties
```

Comment:
Did you want to turn on SQL logging above?

Reply:
We can but it slows things down big time. I want to see if batching gets anything first. Then I can try something else.